fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/Options.java

```diff
@@ -316,6 +322,12 @@ private Builder() {
             optionsMap = Maps.newHashMap();
         }
 
+        @Nonnull
+        public Builder withOptionFromString(Name name, String valueAsString) throws SQLException {
```

Comment:
Did this end up getting used? From what I can tell, the logic in `TestBlock` is letting YAML parse the string into a type